src/com/sun/ts/tests/jpa/core/criteriaapi/From/Client.java

```diff
@@ -1577,7 +1577,7 @@ public void pathGetMapAttributeTest() throws Fault {
 
       getEntityTransaction().begin();
 
-      CriteriaQuery cquery = cbuilder.createQuery(Expression.class);
+      CriteriaQuery cquery = cbuilder.createQuery(Employee.class);
```

Comment:
Looks good to me as pathGetMapAttributeTest() doesn't seem to need the prior use of Expression.class here as per test description which refers to:
> ID	Return	Method/Field	Description	Required	Deprecated	Testable
> ---         ----------    -------------------        ----------------    --------------       -----------------    -----------
> PERSISTENCE:JAVADOC:1098	Expression	jakarta.persistence.criteria.Path.get
( MapAttribute ) 	Create a path corresponding to the referenced map-valued attribute.	true	
	true